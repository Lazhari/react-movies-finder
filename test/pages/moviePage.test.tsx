import React from 'react'
import { render } from '../testUtils'
import Movie from '../../pages/movies/[mid]'

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/movies/464052',
      pathname: '',
      query: '',
      asPath: '',
    }
  },
}))

describe('Movie page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Movie />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
