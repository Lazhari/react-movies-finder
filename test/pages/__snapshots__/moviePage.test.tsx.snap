// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-lg-12"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
